### PR TITLE
WEBEU2 - Preprocessing I - James Eneh 🇳🇬

### PR DESCRIPTION
**Pull request by erozonachi**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,9 +3,11 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (maximesalomon) [inline index.html:6]: Good job adding this line!

File: index.html [hunk 2]
```diff
@@ -3,9 +3,11 @@
...
 
-  <title>My Resume</title>
+  <title>James's Resume</title>
 
+  <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
```
Comment (maximesalomon) [inline index.html:10]: How can you do this with LESS?

File: index.html [hunk 3]
```diff
@@ -14,6 +16,86 @@
...
+                Currently working with Andela as Software Engineer
+              </p>
+            </div>
+          </div>
+        </section>
+      </div>
+    </main>
+    <footer>
+      <h3>Contact</h3>
+      <address>Lagos, Nigeria</address><br>
```
Comment (maximesalomon) [inline index.html:94]: 👍

File: less/index.less [hunk 4]
```diff
@@ -54,6 +54,202 @@ table {
...
+	padding: 1em;
+	border-radius: 1em 0 1em 0;
+
+	@media @mobile-view {
+		border: none;
+		border-radius: none;
+		padding: .5em;
+	}
+
+	header {
```
Comment (maximesalomon) [inline less/index.less:100]: Try to use Prettier to format your code :)

File: less/index.less [hunk 5]
```diff
@@ -54,6 +54,202 @@ table {
...
+				margin-bottom: 1em;
+	
+				img {
+					border: 1em solid @white-color;
+					border-radius: 50%;
+				}
+			}
+	
+			h1 {
+				.flex-display(row, space-evenly, center);
```
Comment (maximesalomon) [inline less/index.less:152]: Good use of Mixin!

**Review by maximesalomon — COMMENTED** [review]
Great job on this James 👍 

- You used variables to set fonts & colors ✅
- You found a good balance with your nesting ✅
- You used mixins ✅
 - You used media queries ✅

- You could create separate LESS files for your variables
- Make sure you read the README fully before starting your projects and use it to check if you have done everything + check the boxes :)

You got this 💪

**Comment by erozonachi** [comment]
Thanks for the feedback @maximesalomon, already reflecting on it 👍🏽

**Pull request merged**